diff --git a/CMakeLists.txt b/CMakeLists.txt
index 70b06c0..ae21c27 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -126,57 +126,25 @@ enable_clang_tidy(On)
 # Get project name
 set(project ${META_PROJECT_NAME})
 
-# Check for system dir install
-set(SYSTEM_DIR_INSTALL FALSE)
-if("${CMAKE_INSTALL_PREFIX}" STREQUAL "/usr" OR "${CMAKE_INSTALL_PREFIX}" STREQUAL "/usr/local")
-    set(SYSTEM_DIR_INSTALL TRUE)
-endif()
-
 # Installation paths
-if(UNIX AND SYSTEM_DIR_INSTALL)
-    # Install into the system (/usr/bin or /usr/local/bin)
-    set(INSTALL_ROOT      "share/${project}")       # /usr/[local]/share/<project>
-    set(INSTALL_CMAKE     "share/${project}/cmake") # /usr/[local]/share/<project>/cmake
-    set(INSTALL_EXAMPLES  "share/${project}")       # /usr/[local]/share/<project>
-    set(INSTALL_DATA      "share/${project}")       # /usr/[local]/share/<project>
-    set(INSTALL_BIN       "bin")                    # /usr/[local]/bin
-    set(INSTALL_SHARED    "lib")                    # /usr/[local]/lib
-    set(INSTALL_LIB       "lib")                    # /usr/[local]/lib
-    set(INSTALL_INCLUDE   "include")                # /usr/[local]/include
-    set(INSTALL_DOC       "share/doc/${project}")   # /usr/[local]/share/doc/<project>
-    set(INSTALL_SHORTCUTS "share/applications")     # /usr/[local]/share/applications
-    set(INSTALL_ICONS     "share/pixmaps")          # /usr/[local]/share/pixmaps
-    set(INSTALL_INIT      "/etc/init")              # /etc/init (upstart init scripts)
-else()
-    # Install into local directory
-    set(INSTALL_ROOT      ".")                      # ./
-    set(INSTALL_CMAKE     "cmake")                  # ./cmake
-    set(INSTALL_EXAMPLES  ".")                      # ./
-    set(INSTALL_DATA      ".")                      # ./
-    set(INSTALL_BIN       ".")                      # ./
-    set(INSTALL_SHARED    "lib")                    # ./lib
-    set(INSTALL_LIB       "lib")                    # ./lib
-    set(INSTALL_INCLUDE   "include")                # ./include
-    set(INSTALL_DOC       "doc")                    # ./doc
-    set(INSTALL_SHORTCUTS "misc")                   # ./misc
-    set(INSTALL_ICONS     "misc")                   # ./misc
-    set(INSTALL_INIT      "misc")                   # ./misc
-endif()
+set(INSTALL_ROOT      "share/${project}")       # /usr/[local]/share/<project>
+set(INSTALL_CMAKE     "share/${project}/cmake") # /usr/[local]/share/<project>/cmake
+set(INSTALL_EXAMPLES  "share/${project}")       # /usr/[local]/share/<project>
+set(INSTALL_DATA      "share/${project}")       # /usr/[local]/share/<project>
+set(INSTALL_BIN       "bin")                    # /usr/[local]/bin
+set(INSTALL_SHARED    "lib")                    # /usr/[local]/lib
+set(INSTALL_LIB       "lib")                    # /usr/[local]/lib
+set(INSTALL_INCLUDE   "include")                # /usr/[local]/include
+set(INSTALL_DOC       "share/doc/${project}")   # /usr/[local]/share/doc/<project>
+set(INSTALL_SHORTCUTS "share/applications")     # /usr/[local]/share/applications
+set(INSTALL_ICONS     "share/pixmaps")          # /usr/[local]/share/pixmaps
+set(INSTALL_INIT      "/etc/init")              # /etc/init (upstart init scripts)
 
 # Set runtime path
 set(CMAKE_SKIP_BUILD_RPATH            FALSE) # Add absolute path to all dependencies for BUILD
 set(CMAKE_BUILD_WITH_INSTALL_RPATH    FALSE) # Use CMAKE_INSTALL_RPATH for INSTALL
 set(CMAKE_INSTALL_RPATH_USE_LINK_PATH FALSE) # Do NOT add path to dependencies for INSTALL
 
-if(NOT SYSTEM_DIR_INSTALL)
-    # Find libraries relative to binary
-    if(APPLE)
-        set(CMAKE_INSTALL_RPATH "@loader_path/../../../${INSTALL_LIB}")
-    else()
-        set(CMAKE_INSTALL_RPATH "$ORIGIN/${INSTALL_LIB}")       
-    endif()
-endif()
-
 
 # 
 # Project modules
@@ -185,24 +153,3 @@ endif()
 add_subdirectory(source)
 add_subdirectory(docs)
 add_subdirectory(deploy)
-
-
-# 
-# Deployment (global project files)
-# 
-
-# Install version file
-install(FILES "${PROJECT_BINARY_DIR}/VERSION" DESTINATION ${INSTALL_ROOT} COMPONENT runtime)
-
-# Install cmake find script for the project
-install(FILES ${META_PROJECT_NAME}-config.cmake DESTINATION ${INSTALL_ROOT} COMPONENT dev)
-
-# Install the project meta files
-install(FILES AUTHORS   DESTINATION ${INSTALL_ROOT} COMPONENT runtime)
-install(FILES LICENSE   DESTINATION ${INSTALL_ROOT} COMPONENT runtime)
-install(FILES README.md DESTINATION ${INSTALL_ROOT} COMPONENT runtime)
-
-# Install runtime data
-if (TARGET cubescape OR TARGET cubescape-log OR TARGET cubescape-qt)
-    install(DIRECTORY ${PROJECT_SOURCE_DIR}/data DESTINATION ${INSTALL_DATA} COMPONENT examples_data)
-endif()
